crates/storage/store.rs

```diff
@@ -459,7 +459,7 @@ impl Store {
                 info!("Received genesis file matching a previously stored one, nothing to do");
                 return Ok(());
             } else {
-                panic!("Tried to run genesis twice with different blocks. Try again after clearing the database. If you're running ethrex as an Ethereum client, run cargo run --release --bin ethrex -- removedb; if you're running ethrex as an L2 run make rm-db-l1 rm-db-l2");
+                return Err(StoreError::Custom(String::from("Tried to run genesis twice with different blocks. Try again after clearing the database. If you're running ethrex as an Ethereum client, run cargo run --release --bin ethrex -- removedb; if you're running ethrex as an L2 run make rm-db-l1 rm-db-l2")));
```

Comment:
remove this stuff `cargo run --release --bin ethrex -- removedb`, or this `make rm-db-l1`. Logs should not have calls to make files. It should be a more generic phrasing, like "remove the database". But I think the messaging should be improved in general. Try to understand the usecase and write a better log.